api/src/main/java/org/openmrs/api/ConceptService.java

```diff
@@ -626,6 +639,8 @@
 	 * @param includeCompleted boolean - include completed proposals as well?
 	 * @return a List&lt;ConceptProposal&gt; object of all found ConceptProposals
 	 * @throws APIException
+	 * @should give a list of all matching concept proposal when include retired is true
+	 * @should give a list of all matching concept proposal when include retired is false
```

Comment:
* should return all concept proposals including retired ones when given true
 * should return all concept proposals excluding retired ones when given false